packages/flutter/lib/src/material/carousel.dart

```diff
@@ -335,6 +438,18 @@ class CarouselView extends StatefulWidget {
   /// The child widgets for the carousel.
   final List<Widget> children;
 
+  /// Called to build carousel item on demand.
+  ///
+  /// Will be called only for indices greater than or equal to zero and less than [itemCount].
+  ///
+  /// Must return a non-null widget for valid indices.
```

Comment:
Below you say:

```
If null, the carousel will continue to build items until [itemBuilder] returns null.
```

Does that contradict these two sentences?

Reply:
Read again, I think my intention was different but updating to make it simple.